upt_macports/upt_macports.py

```diff
@@ -29,6 +31,16 @@ def _render_makefile_template(self):
         template = env.get_template(self.template)
         return template.render(pkg=self)
 
+    @property
+    def licenses(self):
+        path = 'spdx2macports.json'
```

Comment:
It's actually the relative path but since both files are in the same place, it finally is just filename.
Did not want to name it filename as that would be a bit misleading.
How about `relpath` or `relativepath` ?

Reply:
I am not sure that it would be misleading, isn't this always the filename and does ```pkg_resources.resource_filename``` get the (relative) path to this filename? Or am I missing something here (@Steap)?